contracts/crypto-verify/tests/integration.rs

```diff
@@ -337,7 +339,7 @@ fn tendermint_signatures_batch_verify_works() {
     };
 
     let raw = query(&mut deps, mock_env(), verify_msg).unwrap();
-    let res: VerifyResponse = from_slice(&raw).unwrap();
+    let res: VerifyResponse = from_slice(&raw, DESERIALIZATION_LIMIT).unwrap();
```

Comment:
You are changing this API?

Who is going to migrate `cosmwasm-plus` repo? I don't have capacity for many breaking changes now

Reply:
This is changing `cosmwasm_vm::from_slice`, not `cosmwasm_std::from_slice`. As far as I know this is not used in cosmwasm-plus because you are not writing integration tests.

Reply:
Ah, that is good. Sounds fine with me